src/web/forms/DynamicForm.js

```diff
@@ -35,7 +35,7 @@ import {
 } from '../../actions';
 
 import {
-  checkVisibility,
+  referAndGetBool,
```

Comment:
Does 'refer' in this method name refer to doctor referral? Slightly unclear what this method does...